docs/components/toggle/index.html

```diff
@@ -0,0 +1,39 @@
+---
+title: Toggle
+minver: 0.24.0
+also:
+  elements/hx-toggle: <hx-toggle>
+---
+  {% extends 'component.njk' %}
+
+  {% block page_header %}
+    <p>
+      A Toggle is used when a user needs to make a binary decision, such as,
+      turning a setting on or off, or setting a preference enabling or disabling
+      a device or system.
+    </p>
+  {% endblock %}
+
+  {% block content %}
+  <section>
+    <header>
+      <h2 id="basic-checkbox">Toggle</h2>
+      <p></p>
+    </header>
+    <div class="example" id="vue-toggleDemo" v-cloak>
+      <div>
+        <hx-toggle-control class="switch">
```

Comment:
In the same context as the Switch component, we should not need any classes to render the default Toggle component.